test/languages.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+require 'commons/integrity/check/languages.rb'
+require 'commons/builder/config'
+
+describe 'Language consistency' do
+  subject { Commons::Integrity::Check::Languages.new(file, builder_config: builder_config) }
+
+  let(:builder_config) { Config.new('test/fixtures/languages_config.json') }
+
+  describe 'unexpected language codes' do
+    let(:file) { 'test/fixtures/languages_unexpected_popolo.json' }
+
+    it 'finds all errors' do
+      subject.errors.size.must_equal 2
+    end
+
+    it 'names unexpected languages' do
+      subject.errors.map(&:message).must_include 'Unexpected language: en_US'
+      subject.errors.map(&:message).must_include 'Unexpected language: zh_TW'
```

Comment:
In this repo I'd quite like us to follow the Single Expectation principle: http://www.betterspecs.org/#single